src/ansys/edb/core/session.py

```diff
@@ -503,16 +509,26 @@ def session(ansys_em_root, port_num, ip_address=None):
     >>> with session("C:\\Program Files\\AnsysEM\\v231\\Win64", 50051):
     >>>    # program goes here
     """
+    reuse_session = False
+    if MOD.current_session is not None:
```

Comment:
@chenchienjacklin Can you refactor this logic into a helper method that can be used here and in `launch_session(...)`?